packages/next/build/webpack-config.ts

```diff
@@ -1529,6 +1530,14 @@ export default async function getBaseWebpackConfig(
 
   const webpack5Config = webpackConfig as webpack5.Configuration
 
+  webpack5Config.module!.rules!.unshift({
+    test: /\.wasm$/,
```

Comment:
Do we want to prefix this? Otherwise its going to change all wasm imports instead of just middleware, right?

Reply:
I think the secret sauce is `issuerLayer` which should only affect the middleware layer